### DISCUSSION TITLE
Cannot apply for GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by Seanji2002** [community/community#29881] [GitHub Education]
I cannot apply for GitHub Student Developer Pack for some reason even if I uploaded my transcript/proof of enrollment and use my school email as an account. I've attempted for like 10 times already. Please help 😢 . It also says that I have the location turned off even if I have it on :(.

**Comment by melvincwng** [comment]
@Seanji2002, please kindly raise a support ticket over [here](https://support.github.com/contact/education) and provide some time for the GitHub team to investigate the issue.

Hope this helps.

**Reply by Seanji2002** [reply]
Thank you!
